03numbersWithOnly1InBinary.cpp

```diff
@@ -0,0 +1,50 @@
+#include <iostream>
+#include <cmath>
+
+using namespace std;
+
+int fromBinaryToDecimal(int binaryNumber);
+void printOnly1ContainBinary(int decimalNumberEntered);
+
+int main()
+{
+	int decimalNumberEntered;
+
+	cout << "This program will show you all decimal numbers from 0 to desired which consist of only 1 if converted to binary numbers." << endl << endl;
+
+	cout << "Enter desired decimal number: ";
+	cin >> decimalNumberEntered;
+
+	printOnly1ContainBinary(decimalNumberEntered);

```

Comment:
the same

Reply:
![image](https://user-images.githubusercontent.com/79978383/111069305-7a559500-84dd-11eb-8b21-9b81d78a5699.png)
Done

Reply:
![image](https://user-images.githubusercontent.com/79978383/111069330-9bb68100-84dd-11eb-8b8d-2b3cd7a9b8d8.png)